docs/getting_started/installation_cn.md

```diff
@@ -0,0 +1,96 @@
+# 安装
+
+### 使用PIP安装
+Visual DL提供独立的Python SDK，如果训练任务是基于Python的话，直接安装visualdl的whl包，import到自己项目中即可使用。
+
+```
+pip install --upgrade visualdl
+```
+
+如果以上步骤出现问题，很可能是因为python或pip不同版本或不同位置所致，以下安装方法能解决。
+
+## 使用 virtualenv 安装
```

Comment:
Should we move away from 'virtualenv' towards Anaconda's 'conda create' ? Because we tried 'virtualenv' but got some issues, right?

Reply:
That's valid point. In general we need to understand what exactly causes the issue. 

I don't think this PR is suitable to address this. Let's sit down together and research more on this topic.